jbpm-human-task/jbpm-human-task-audit/src/main/java/org/jbpm/services/task/audit/JPATaskLifeCycleEventListener.java

```diff
@@ -196,9 +196,8 @@ public void afterTaskStoppedEvent(TaskEvent event) {
         }
 
     }
-
     @Override
-    public void afterTaskCompletedEvent(TaskEvent event) {
+    public void beforeTaskCompletedEvent(TaskEvent event) {
```

Comment:
not sure why about this change? shouldn't we keep on updating the tasks after task completion?

Reply:
cannot happen as otherwise tast after completed will be executed after the process is finished causing the events not happening in the correct order.